bug: Converting from USD to ETH does not work when creating a new bounty

<!--
Hello Gitcoiner!

Please use the template below for bugs found within Gitcoin.
If it is general support you need, reach out to us on [Discord](https://discord.gg/gitcoin)

Provide a general summary of the issue in the title above and use relevant fields below to define the problem.

-->

**Describe the bug**
```
raven.js:58 Error: [ethjs-unit] while converting number 0.0018880000000000001 to wei, too many decimal places
    at Object.toWei (web3.min.fbcef5f40431.js:1)
    at Object.toWei (web3.min.fbcef5f40431.js:1)
    at new_bounty.fcedd9fdd8f5.js:432
    at new Promise (<anonymous>)
    at wn.payFees (new_bounty.fcedd9fdd8f5.js:426)
    at wn.submitForm (new_bounty.fcedd9fdd8f5.js:484)
    at click (eval at Ya (vue.c107ca5cd927.js:1), <anonymous>:3:48871)
    at He (vue.c107ca5cd927.js:6)
    at HTMLButtonElement.n (vue.c107ca5cd927.js:6)
    at HTMLButtonElement.Yr.o._wrapper (vue.c107ca5cd927.js:6)
    at HTMLButtonElement.d (raven.js:445)
```

**To Reproduce**
1. Go to https://gitcoin.co/bounty/new
2. Select ETH as denomination, and type 50 into the USD field.
3. Submit form (nothing happens when pressing the button).
  <img width="1440" alt="Screen Shot 2021-06-02 at 12 43 08 PM" src="https://user-images.githubusercontent.com/15718174/120468805-4b4b1300-c3aa-11eb-8937-f48aff2551f1.png">

--- a/app/assets/v2/js/pages/new_bounty.js
+++ b/app/assets/v2/js/pages/new_bounty.js
@@ -437,7 +437,7 @@ Vue.mixin({
           web3.eth.sendTransaction({
             to: toAddress,
             from: selectedAccount,
-            value: web3.utils.toWei(String(vm.totalAmount.totalFee), 'ether')
+            value: web3.utils.toWei(new web3.utils.BN(BigInt(vm.totalAmount.totalFee.toFixed(18) * 1.0 * Math.pow(10, 18))).toString(), 'wei')
           }).once('transactionHash', (txnHash, errors) => {
 
             console.log(txnHash, errors);

--- a/app/assets/v2/js/pages/new_bounty.js
+++ b/app/assets/v2/js/pages/new_bounty.js
@@ -437,7 +437,7 @@ Vue.mixin({
           web3.eth.sendTransaction({
             to: toAddress,
             from: selectedAccount,
-            value: web3.utils.toWei(new web3.utils.BN(BigInt(vm.totalAmount.totalFee.toFixed(18) * 1.0 * Math.pow(10, 18))).toString(), 'wei')
+            value: BigInt(vm.totalAmount.totalFee.toFixed(18) * Math.pow(10, 18)).toString()
           }).once('transactionHash', (txnHash, errors) => {
 
             console.log(txnHash, errors);

